src/pages/Contactpage.jsx

```diff
@@ -1,5 +1,54 @@
+import { useState } from "react";
 import PageNav from "../components/PageNav";
+
 function Contactpage() {
+  const [formData, setFormData] = useState({
+    email: "",
+    subject: "",
+    message: "",
+  });
+
+  const [statusMessage, setStatusMessage] = useState("");
+
+  // Handle form input change
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.id]: e.target.value,
+    });
+  };
+
+  // Function to submit form data to the API
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    // Create payload to send to the backend
+    const payload = {
+      subject: formData.subject,
+      email: formData.email,
+      message: formData.message,
+    };
+
+    try {
+      const response = await fetch("http://localhost:8080/notification/email", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(payload),
```

Comment:
You don't have to stringify the payload.
In the backend, you are reading JSON data but stringify makes json into string.

Reply:
> You don't have to stringify the payload. In the backend, you are reading JSON data but stringify makes json into string.

![image](https://github.com/user-attachments/assets/606ac846-00b5-4446-97e0-20fdbd8d98e7)
we are using express middleware to convert the Raw json data to json object that's why sending string.